rust-runtime/aws-smithy-http-server-python/examples/pokemon_service.py

```diff
@@ -229,7 +244,9 @@ async def stream_pokemon_radio(_: StreamPokemonRadioInput, context: Context):
 ###########################################################
 # Run the server.
 ###########################################################
-def main():
+def main() -> None:
     app.run(workers=1)
```

Comment:
Do we need a separate `main` function? 🤔

Reply:
I think it is not required but having a explicitly defined [entry point](https://packaging.python.org/en/latest/specifications/entry-points/) is kinda useful when you want to setup your service via a supervisor or something